src/org/usfirst/frc4904/robot/commands/HopperSetBallio.java

```diff
@@ -0,0 +1,10 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.subsystems.Hopper;
+
+public class HopperSetBallio extends HopperSet {
```

Comment:
Should this be `HopperSetBallIO` for consistency? There's inconsistency across the code, so I don't know how much it matters at this point.